pkg/driver/controller.go

```diff
@@ -42,11 +42,15 @@ const (
 
 	volumeParamsSubnetId                 = "subnetId"
 	volumeParamsSecurityGroupIds         = "securityGroupIds"
+	volumeParamsAutoImportPolicy         = "autoImportPolicy"
 	volumeParamsS3ImportPath             = "s3ImportPath"
 	volumeParamsS3ExportPath             = "s3ExportPath"
 	volumeParamsDeploymentType           = "deploymentType"
 	volumeParamsKmsKeyId                 = "kmsKeyId"
 	volumeParamsPerUnitStorageThroughput = "perUnitStorageThroughput"
+	volumeParamsAutomaticBackupRetentionDays   = "automaticBackupRetentionDays"
```

Comment:
I would suggest you fmt these lines. Same as above

Reply:
fixed